lib/srv/db/mysql/engine.go

```diff
@@ -38,30 +37,26 @@ import (
 	"github.com/siddontang/go-mysql/server"
 
 	"github.com/gravitational/trace"
-	"github.com/jonboulle/clockwork"
 	"github.com/sirupsen/logrus"
 )
 
+func init() {
+	common.RegisterEngine(defaults.ProtocolMySQL,
+		func(ec common.EngineConfig) (common.Engine, error) {
+			return &Engine{
+				EngineConfig: ec,
+			}, nil
+		})
+}
```

Comment:
nit: What would you say about: 
```suggestion
func init() {
	common.RegisterEngine(defaults.ProtocolMySQL, newEngine)
}

func newEngine(ec common.EngineConfig) (common.Engine, error} {
	return &Engine{
		EngineConfig: ec,
	}, nil
}
```

Reply:
This looks cleaner, yeah. Updated.